examples/wallet/app/utils/wasmWrapper.js

```diff
@@ -147,47 +122,91 @@ export async function buildTransaction(
   } = await wasmBindings;
   const { calculateFee } = feeCalculator(nodeSettings);
   const privateKey: PrivateKey = PrivateKey.from_bech32(secret);
-  const sourcePublicKey: PublicKey = privateKey.to_public();
-  const sourceAccount: Account = Account.from_public_key(sourcePublicKey);
+  const sourceAccount: Account = Account.from_public_key(
+    privateKey.to_public()
+  );
 
-  // 1 input + 1 output
-  const computedFee = calculateFee(1, 1, 0);
-  const inputAmount = computedFee + amount;
+  // 1 input + 1 certificate
```

Comment:
what does the `1 certificate` part mean?

Reply:
It means that I copypasted it from the `buildDelegationTransaction` :upside_down_face: